docs/installation/server-config/security.rst

```diff
@@ -0,0 +1,163 @@
+.. _ksql-security:
+
+Securing KSQL
+-------------
+
+To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. In the same way,
+KSQL is configured so that the KSQL servers are authenticated and authorized, and data communication is encrypted when
+communicating with the Kafka cluster. KSQL can be configured for:
+
+- :ref:`SSL for encryption <kafka_ssl_encryption>`
+- :ref:`SASL for authentication <kafka_sasl_auth>`
+- :ref:`HTTPS for Confluent Schema Registry <schema_registry_http_https>`
+
+-------------------------------------------------
+Using KSQL with an Apache Kafka Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Interactive KSQL clusters require that the KSQL user has open access to create, read, write and delete topics and
+use any consumer group.
+
+The required ACLs are:
+
+- *DESCRIBE_CONFIGS* permission on the *CLUSTER*.
+- *CREATE* permission on the *CLUSTER*.
+- DESCRIBE*, *READ*, *WRITE* and *DELETE* permissions on the *<any>* *TOPIC*.
+- *DESCRIBE* and *READ* permissions  on the *<any>* *GROUP*.
+
+It is still possible to restrict the KSQL user from accessing specific resources using *DENY* ACLs. For example, you can add a
+*DENY* ACL to stop KSQL queries from accessing a topic that contains sensitive data.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Non-Interactive KSQL clusters
```

Comment:
We're still deciding on terminology, but for now let's go with:

> Non-Interactive (headless) KSQL clusters

Reply:
with `(headless)` included ^

Reply:
@miguno regarding the term `headless` - why not just use existing terminology around CRUD and call these `Read-only KSQL Clusters`?

Reply:
-1 on "read-only".  Particularly because the KSQL clusters will still read+write data.

Reply:
When ever I see `headless` it just makes me think..

> Non-Interactive (headless) chicken.

:)